calblueprint/vouchers4veggies
# Selene/login screen

# ✨ New in this PR ✨

## One liner: what did you do? 🚨
Created login screen based on Figma mockup.

## How can the reviewer test your code? 👩‍💻
Run `npx expo start --web` and check login screen.
In `App.tsx`, uncomment the line `return <StartScreen></StartScreen>;` and check start screen.

## Any bugs you encountered or still having trouble with? 🐛

- According to the [documentation](https://reactnative.dev/docs/button#color), the button color sets background for Android and text color for IOS. I feel like there should be a way to specify both, this seems like it will be a problem?
- Navigation has not been implemented.

## Resources 📔
[Initial screens PR](https://github.com/calblueprint/vouchers4veggies/pull/1)
